### PR TITLE
Added font support for SVG text

### PR DESCRIPTION
**Pull request by DrGabble**
Fixes https://github.com/linebender/druid/issues/1827

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ You can find its changes [documented below](#070---2021-01-01).
 
 ### Added
 
+- System fonts loaded so that SVG images render text ([#1850] by [@DrGabble])
```
Comment (cmyr) [inline CHANGELOG.md:15]: you'll also need to add the link to the PR below. :)

**Review by cmyr — APPROVED** [review]
This seems reasonable. It would be nice if we could somehow defer font loading until we knew that fonts were needed, since it likely has some cost, but I'm fine with worrying about that later if it seems like a problem. :)

**Comment by DrGabble** [comment]
> This seems reasonable. It would be nice if we could somehow defer font loading until we knew that fonts were needed, since it likely has some cost, but I'm fine with worrying about that later if it seems like a problem. :)

Yeah doing that automatically is probably hard. Could add a function for "new_with_text" but seems like people might miss it. Agreed it will make things more expensive but we can fix that later if it becomes an issue :)

File: CHANGELOG.md (hunk 1 above)
Comment (cmyr) [reply CHANGELOG.md:15]: @DrGabble this is ready to merge except for this missing link!

**Comment by DrGabble** [comment]
Yes sorry, VM packed in, it's on my to-do list don't worry

Get Outlook for Android<https://aka.ms/AAb9ysg>

________________________________

File: CHANGELOG.md (hunk 1 above)
Comment (DrGabble) [reply CHANGELOG.md:15]: @cmyr done!

**Pull request merged**